org-formation/070-guard-duty/_tasks.yaml

```diff
@@ -22,32 +22,12 @@ GuardDuty:
     LogArchiveBinding:
       Account: !Ref LogCentralAccount
     MemberBinding:
-      Account:
-        - !Ref TransitAccount
-        - !Ref AdminCentralAccount
-      #  - !Ref LogCentralAccount
-        - !Ref ImageCentralAccount
-      #  - !Ref SecurityCentralAccount
-      OrganizationalUnit:
-        - !Ref BridgeOU
-        - !Ref ServiceCatalogOU
-        - !Ref ScienceDevOU
-        - !Ref ScienceProdOU
-        - !Ref SynapseOU
+      Account: '*'
       IncludeMasterAccount: true
+      ExcludeAccount:
+        - !Ref accountId
```

Comment:
This is an explicit exclude of the securitycentral account.  Can you confirm the `IncludeMasterAccount: true` binding will enforce GD on securitycentral and that the organizations master account is included in the `*` member binding?

Reply:
to confirm accounts that have guardduty enabled just goto securitycentral -> guardduty -> accounts

Reply:
confirmed, zaro manually ran this config first, and the state is as expected except for `strides-ampad-workflows`